.maintain/monitoring/healthcheck-lambda.js

```diff
@@ -0,0 +1,185 @@
+const https = require('https');
+const http = require('http');
+const url = require('url');
+const webhookUrl = process.env.SLACK_NOTIFCATION_WEBHOOK_URL;
+const port = 9933;
+
+async function hasErrorStatus(status) {
+  if (status.result && status.result.startsWith('0x')) return false;
```

Comment:
`return !(status.result && status.result.startsWith('0x'))` as function body is sufficient